rest_framework/templates/rest_framework/base.html

```diff
@@ -2,33 +2,43 @@
 {% load rest_framework %}
 {% load i18n %}
 
+
+{% block prehead %}
```

Comment:
We want to allow this to be overridden, no? As I explained, then I can include this template into an existing base template.